README.md

```diff
@@ -1,47 +1,56 @@
 # Mantle: Gluing CoreOS together
 
-This repository is a collection of utilities for the CoreOS SDK.
-
-## plume
-
-CoreOS release utility
-
-## plume index
-
-Generate and upload index.html objects to turn a Google Cloud Storage
-bucket into a publicly browsable file tree. Useful if you want something
-like Apache's directory index for your software download repository.
+This repository is a collection of utilities for the Container Linux SDK. Most of the
```

Comment:
I'm not requesting changing this now since it was in the original text, but in my opinion the phrasing that these tools are "for the SDK" is wrong.  Most of these tools have little-to-nothing to do with the SDK (`cork` being the big exception), and our release process runs them all outside the SDK (except `kola` for QEMU).

Also, a newer program `gangue` is not described in the document.  It is available in the SDK and developer images as a portage URL handler to allow cryptographic verification of Gentoo binary packages.  Not sure if that warrants inclusion in the README or not.